internal/credentials/file.go

```diff
@@ -207,6 +207,12 @@ func (c *fileCredentialsService) Delete(guid string) error {
 	return nil
 }
 
+func (c *fileCredentialsService) Reset() error {
+	c.log.Warn("Corrupted credentials data found. A reset was applied to stored data to be able to proceed.", "credentials_service", "file")
```

Comment:
We will have a different behavior here anyway (see my comment on the PR), so this suggestion is a little moot, but I'm going to make it anyway since precisions and conciseness are skills and seeing edits is one way to learn that

```suggestion
	c.log.Warn("Corrupted credentials data found. The stored data was reset.", "credentials_service", "file")
```